cargo-near/src/commands/abi/mod.rs

```diff
@@ -3,10 +3,10 @@ use cargo_near_build::abi::AbiOpts;
 #[derive(Debug, Clone, interactive_clap::InteractiveClap)]
 #[interactive_clap(input_context = near_cli_rs::GlobalContext)]
 #[interactive_clap(output_context = AbiCommandlContext)]
-pub struct AbiCommand {
-    /// disable implicit `--locked` flag for all `cargo` commands, enabled by default
+pub struct Command {
+    /// enable implicit `--locked` flag for all `cargo` commands, disabled by default
```

Comment:
"implicit" looks unnecessary and confusing

```suggestion
    /// enable `--locked` flag for all `cargo` commands, disabled by default
```